packages/core/src/session_pool/session_pool.ts

```diff
@@ -306,6 +314,11 @@ export class SessionPool extends EventEmitter {
         await this.persistState();
     }
 
+    /** @internal */
+    getBlockedStatusCodes(additionalCodes: number[] = []): number[] {
+        return this.blockedStatusCodes.concat(additionalCodes);
+    }
+
```

Comment:
Can we get rid of this somehow? It's a super weird function. Why would I want to get status codes and provide additional codes to the function to concat them?

Reply:
I did it this way because I thought users might be creating the `Session` class manually, and wanted to have it working for them too. We have the config instance, so we can make this a config option instead, and read it from there. That way we dont need this method at all and it will be configurable via the JSON file as well.